drivers/flash/flash_simulator.c

```diff
@@ -48,7 +48,7 @@
 #error "Erase unit must be a multiple of program unit"
 #endif
 
-#define MOCK_FLASH(addr) (mock_flash + (addr) - FLASH_SIMULATOR_BASE_OFFSET)
+#define MOCK_FLASH(rel_addr) (mock_flash + (rel_addr))
```

Comment:
I am not sure `rel` means. It should probably be a full word.

Reply:
@jeppenodgaard 

You're right. This is not a good name.

Is it ok, if I just name this to `offset` which is the same name that is used elswhere in the file for the same thing?